lib/messagebird/client.rb

```diff
@@ -51,16 +53,16 @@ def request(method, path, params={})
       # Parse the HTTP response.
       case response.code.to_i
       when 200, 201, 204, 401, 404, 405, 422
-        json = JSON.parse(response.body)
+        unless response.body.nil?
+          json = JSON.parse(response.body)
+          # If the request returned errors, create Error objects and raise.
+          if json.has_key?('errors')
+            raise ErrorException, json['errors'].map { |e| Error.new(e) }
+          end
+        end
```

Comment:
if the body is null the `json` wont be defined when we return. Maybe we need to give it a default value.